src/dioptra/restapi/v1/artifacts/controller.py

```diff
@@ -46,6 +47,7 @@
 from .service import (
     RESOURCE_TYPE,
     SEARCHABLE_FIELDS,
+    ArtifactIdContentsService,
```

Comment:
causes an import error right now. I realize you haven't worked on the service layer yet, but I found it useful to create a class stub, so flask would run and I could view the endpoint and schema in swagger.